packages/shared/useCounter/index.ts

```diff
@@ -12,8 +15,9 @@ export interface UseCounterOptions {
  * @param [initialValue=0]
  * @param {Object} options
  */
-export function useCounter(initialValue = 0, options: UseCounterOptions = {}) {
-  const count = ref(initialValue)
+export function useCounter(initialValue: MaybeRef<number> = 0, options: UseCounterOptions = {}) {
+  let _initialValue = toValue(initialValue)
+  const count = toRef(initialValue)
```

Comment:
Just a thought, if we only receive a `MaybeRef` here, is it appropriate to utilize `ref` and `unref`?

```ts
export function useCounter(initialValue: MaybeRef<number> = 0, options: UseCounterOptions = {}) {
  let _initialValue = unref(initialValue)
  const count = ref(initialValue)

  // ...
}
```

Or let `initialValue` support `MaybeRefOrGetter<number>`, but the getter here is not the getter of `computed`, it is only a function to get the initial value.

Reply:
Thank for your advise, I think it's more appropriate for using `ref` & `unref `